### PR TITLE
Structured log migration for pkg/util/netsh

### PR DESCRIPTION
**Pull request by calvin0327**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:

ref:  
* [Structured Logging migration instructions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md)

#### Special notes for your reviewer:

This is a supplement for PR #99529.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by pacoxu** [comment]
/triage accepted
/sig instrumentation
/lgtm
/assign @caseydavenport
for approval

File: pkg/util/netsh/netsh.go [hunk 1]
```diff
@@ -114,12 +114,12 @@ func (runner *runner) EnsureIPAddress(args []string, ip net.IP) (bool, error) {
...
 	exists, _ := checkIPExists(ipToCheck, argsShowAddress, runner)
 	if exists == true {
-		klog.V(4).InfoS("not adding IP address, as it already exists", "ipToCheck", ipToCheck)
+		klog.V(4).InfoS("Not adding IP address, as it already exists", "ipToCheck", ipToCheck)
 		return true, nil
 	}
 
 	// IP Address is not already added, add it now
-	klog.V(4).InfoS("running netsh interface ipv4 add address", "address", args)
+	klog.V(4).InfoS("Running netsh interface ipv4 add address", "address", args)
```
Comment (aojea) [inline pkg/util/netsh/netsh.go:122]: the docs says to use IP , uppercase everywhere https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

There is also one "ip" as key in Lines 130

File: pkg/util/netsh/netsh.go [hunk 2]
```diff
@@ -66,7 +66,7 @@ func New(exec utilexec.Interface) Interface {
 
 // EnsurePortProxyRule checks if the specified redirect exists, if not creates it.
 func (runner *runner) EnsurePortProxyRule(args []string) (bool, error) {
-	klog.V(4).InfoS("running netsh interface portproxy add v4tov4", "args", args)
+	klog.V(4).InfoS("Running netsh interface portproxy add v4tov4", "args", args)
```
Comment (aojea) [inline pkg/util/netsh/netsh.go:69]: what is the best key "args" or "arguments", or just not key at all like in the example for httplog in https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Comment (calvin0327) [reply pkg/util/netsh/netsh.go:69]: Yes, I think the "arguments" is more suitable。

**Comment by mengjiao-liu** [comment]
Thanks for catch!
/lgtm

**Comment by aojea** [comment]
/lgtm
/approve
Thanks

**Pull request merged**